nevergrad/functions/gym/gym_multi.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import os
+import gym
+
+if os.name != "nt":
+    import gym_anm  # type: ignore  # pylint: disable=unused-import
```

Comment:
```suggestion
    import gym_anm  # pylint: disable=unused-import
```
type ignore should not be necessary since it is already ignore in mypy.ini